service/src/main/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleService.java

```diff
@@ -89,4 +89,9 @@ private LDUser createLDUser(Map<String, LDValue> values) {
         values.forEach(builder::custom);
         return builder.build();
     }
+
+    public boolean isServeOrdersAndDocsToOthersEnabled() {
```

Comment:
can this go above the private methods, if you want to ensure no merge conflicts then just stick it in the middle of the class instead of at the end of all the public methods